chat-tutorial/README.md

```diff
@@ -0,0 +1,21 @@
+# Chat Tutorial
+
+Visual Studio Code's Copilot Chat architecture enables extension authors to integrate with the GitHub Copilot Chat experience. A chat extension is a VS Code extension that uses the Chat extension API by contributing a Chat participant. Chat participants are domain experts that can answer user queries within a specific domain.
+
+This GitHub Copilot Extension sample shows:
+
+- How to contribute a chat participant to the GitHub Copilot Chat view.
+
+![demo](./demo.png)
```

Comment:
This image file doesn't exist. Also, if you're going to provide a demo image, please also provide a meaningful alt-text to help with accessibility. Thank you. 🤓

Reply:
good catch, forgot to move the image over. updated alt text too!